bower.json

```diff
@@ -24,10 +24,11 @@
     "iron-a11y-announcer": "polymerelements/iron-a11y-announcer#^1.0.0"
   },
   "devDependencies": {
-    "iron-component-page": "polymerelements/iron-component-page#^1.0.0",
-    "paper-button": "polymerelements/paper-button#^1.0.0",
     "web-component-tester": "*",
```

Comment:
This is orthogonal to this change, but could you do me a favor and update this to: `"web-component-tester": "polymer/web-component-tester#^3.4.0"`

:bow: